File: SER321-master/Assigns/Assign2/src/cpp/MovieDescription.cpp
#include "MovieDescription.hpp"
#include <string>
#include <iostream>
using namespace std;

MovieDescription::MovieDescription(string newTitle, string newRated, string newReleased, string newRuntime, string newPlot, string newFilename, string newGenre, string newActors)
{
  setTitle(newTitle);
  setRated(newRated);
  setReleased(newReleased);
  setRuntime(newRuntime);
  setPlot(newPlot);
  setFilename(newFilename);
  setGenre(newGenre);
  setActors(newActors);
}

void MovieDescription::setTitle(string newTitle)
{
  title = newTitle;
}

void MovieDescription::setRated(string newRated)
{
  rated = newRated;
}

void MovieDescription::setReleased(string newReleased)
{
  released = newReleased;
}

void MovieDescription::setRuntime(string newRuntime)
{
  runtime = newRuntime;
}

void MovieDescription::setPlot(string newPlot)
{
  plot = newPlot;
}

void MovieDescription::setFilename(string newFilename)
{
  filename = newFilename;
}

void MovieDescription::setGenre(string newGenre)
{
  genre = newGenre;
}

void MovieDescription::setActors(string newActors)
{
  actors = newActors;
}

string MovieDescription::getTitle()
{
  return title;
}

string MovieDescription::getRated()
{
  return rated;
}

string MovieDescription::getReleased()
{
  return released;
}

string MovieDescription::getRuntime()
{
  return runtime;
}

string MovieDescription::getPlot()
{
  return plot;
}

string MovieDescription::getFilename()
{
  return filename;
}

string MovieDescription::getGenre()
{
  return genre;
}

string MovieDescription::getActors()
{
  return actors;
}

File: SER321-master/Assigns/Assign2/src/java/MovieDescription.java
public class MovieDescription
{
  private String title;
  private String rated;
  private String released;
  private String runtime;
  private String plot;
  private String filename;
  private String genre;
  private String actors;

  public MovieDescription(String title, String rated, String released, String runtime, String plot, String filename, String genre, String actors)
  {
    setTitle(title);
    setRated(rated);
    setReleased(released);
    setRuntime(runtime);
    setPlot(plot);
    setFilename(filename);
    setGenre(genre);
    setActors(actors);
  }

  public void setTitle(String title)
  {
    this.title = title;
  }

  public void setRated(String rated)
  {
    this.rated = rated;
  }

  public void setReleased(String released)
  {
    this.released = released;
  }
  public void setRuntime(String runtime)
  {
    this.runtime = runtime;
  }

  public void setPlot(String plot)
  {
    this.plot = plot;
  }

  public void setFilename(String filename)
  {
    this.filename = filename;
  }

  public void setGenre(String genre)
  {
    this.genre = genre;
  }
  public void setActors(String actors)
  {
    this.actors = actors;
  }

  public String getTitle()
  {
    return title;
  }

  public String getRated()
  {
    return rated;
  }

  public String getReleased()
  {
    return released;
  }

  public String getRuntime()
  {
    return runtime;
  }

  public String getPlot()
  {
    return plot;
  }

  public String getFilename()
  {
    return filename;
  }

  public String getGenre()
  {
    return genre;
  }

  public String getActors()
  {
    return actors;
  }
}
